pkg/provider/provider.go

```diff
@@ -1,10 +1,12 @@
 package provider
 
-import "regexp"
+import (
+	"regexp"
+)
 
-const (
-	retestRegex   = "(^|\\\\r\\\\n)/retest([ ]*$|$|\\\\r\\\\n)"
-	oktotestRegex = "(^|\\\\r\\\\n)/ok-to-test([ ]*$|$|\\\\r\\\\n)"
+var (
+	retestRegex   = regexp.MustCompile(`(?m)^/retest\s*$`)
```

Comment:
I think that would make: 

```
/lgtm

good to go
```

work

but not : 

```
good to go

/lgtm
```

Reply:
you mean
```
{
			name:    "valid comments",
			comment: "/lgtm \n/retest \n/approve",
			want:    true,
},
```
we have a test

Reply:
no i mean a sentence first and a slash command (ie: /retest) after

Reply:
```
good to go
/retest
```
so this is invalid?
as per current logic if `/retest` is there in the comment, it should be a separate line with no other text on that line
then it pass

Reply:
okay if that works 👍🏻